src/components/Comments.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react";
+import fetchComments from "../controllers/fetchComments";
+import CommentCard from "./CommentCard";
+
+function Comments({ articleId }) {
+  const [comments, setComments] = useState([]);
+  useEffect(() => {
+    const getComments = async () => {
+      const comments = await fetchComments(articleId);
+      setComments(comments);
+    };
+    getComments();
+  }, []);
+  const rtf1 = new Intl.RelativeTimeFormat("en", { style: "short" });
+  return (
+    <>
+      {comments.map((comment) => {
```

Comment:
We should handle the case where an article exists but there are no comments to show just yet